src/index.js

```diff
@@ -21,6 +22,15 @@ const publicDirPath = path.join(__dirname, '../public')
 
 app.use(express.static(publicDirPath))
 
+app.get('/rooms', async (req, res) => {
+    try {
+        const rooms = getRooms()
+        res.status(200).send(rooms)
```

Comment:
for future: it's batter not send `{rooms}`. It allows you in future add other data to response, for instance pagination
```javascript
res.json({ rooms, meta: { page, totalPages, count})
```

Reply:
I understand it's 'for future', but I think I fixed it.
I changed the endpoint to this: `res.send({ rooms })`.
And I changed the client to handle this response to this: `const { rooms } = await response.json().`

OR
Would it be better to keep the client line I changed as it was `(`const rooms = await response.json().`)`.
And instead change this line: `const html = Mustache.render(roomsTemplate.innerHTML, { rooms: rooms.rooms })`?

The first was seems a bit more convenient to use in case more than one property is passed, cuz that way I could just extract whatever I need and use by its name, instead of rooms.rooms.blah.blah.blah.